libs/payments/ui/src/lib/nestapp/app.module.ts

```diff
@@ -49,7 +49,7 @@ import { StatsD } from 'hot-shots';
       useValue: {}, //Temporary value to resolve Payments Next startup issues
     },
     {
-      provide: StatsD,
+      provide: StatsDService,
```

Comment:
This change was necessary to fix an issue preventing `payments-next` from starting up.